testsuite/ext-tools/json_creation_script.py

```diff
@@ -0,0 +1,168 @@
+import argparse
```

Comment:
Let's rename it, what about... "maintenance_json_generator.py" ?

Reply:
renamed